Bottom: e55906c0e954fbf21de11f857cb3a31e10187813
Top:    e49f9113a771d628cb53aa5bc56b67bbdfbc87df
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-06-05 12:05:33 +0100

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index af5aa3e..70cc17f 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3090,7 +3090,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
